### PR TITLE
the Hakkari are now considered evil by every order and life faction

### PR DESCRIPTION
**Pull request by Mihkelval**
## Changelog:
- The Hakkari faith is now considered evil by every other order and life faith.

### CODE REVIEW

File: common/religion/doctrines/wc_doctrines_religions.txt [hunk 1]
```diff
@@ -58,6 +58,7 @@
 			hostility_override_tenet_life_syncretism = 1
 			
 			#Pure evil religions
+			hostility_override_intolerance_hostility_doctrine = 3
```
Comment (zumbak04) [inline common/religion/doctrines/wc_doctrines_religions.txt:61]: I would extend it to other religious families to make sure everyone considers the Hakkari to be evil.

**Comment by zumbak04** [comment]
Faiths with `intolerance_hostility_doctrine` shouldn't be able to pick syncretism tenets because it won't change anything.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
`hostility_override_intolerance_hostility_doctrine` isn't not localized. Localize it like this: `doctrine_parameter_hostility_override_intolerance_hostility_doctrine:0 ""`. Thus it will be hidden empty line.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/religion/doctrines/wc_doctrines_religions.txt [hunk 2]
```diff
@@ -102,12 +115,14 @@
...
 			hostility_same_family = 1
-			hostility_others = 3
+			hostility_others = 2
 			
 			#Syncretic religions
 			hostility_override_tenet_shadow_syncretism = 1
 			
 			#Pure evil religions
+			hostility_override_intolerance_hostility_doctrine = 3
+			doctrine_parameter_hostility_override_intolerance_hostility_doctrine:0 ""
```
Comment (zumbak04) [inline common/religion/doctrines/wc_doctrines_religions.txt:125]: Wrong place. It must be added in loc file like `localization/english/religion/wc_core_tenets_l_english.yml`.

File: common/trigger_localization/wc_doctrine_triggers.txt [hunk 3]
```diff
@@ -14,4 +14,8 @@ incompatible_tenet_consumerism_trigger = {
 incompatible_tenet_primitivism_trigger = {
 	first_not = INCOMPATIBLE_TENET_PRIMITIVISM_TRIGGER
 	third_not = INCOMPATIBLE_TENET_PRIMITIVISM_TRIGGER
+}
+incompatible_tenet_intolerance_trigger{
```
Comment (zumbak04) [inline common/trigger_localization/wc_doctrine_triggers.txt:18]: `=` is missing.
```suggestion
incompatible_tenet_intolerance_trigger = {
```

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/doctrines/wc_doctrines_religions.txt (hunk 1 above)
Comment (Mihkelval) [reply common/religion/doctrines/wc_doctrines_religions.txt:68]: done

File: common/religion/doctrines/wc_doctrines_religions.txt (hunk 2 above)
Comment (Mihkelval) [reply common/religion/doctrines/wc_doctrines_religions.txt]: done, moved them all to the loc file

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**